### ISSUE TITLE
[A11y] Screen Reader does not announce opened link balloon

### ISSUE THREAD

**Issue by pkliesch** [ckeditor/ckeditor5#16521]
## 📝 Provide detailed reproduction steps (if any)

1. Create a link in the editor.
2. Start your screen reader / VoiceOver
3. Close the balloon and navigate the cursor onto the created link
4. Press [Ctrl] + K to open the link balloon

### ✔️ Expected result

The screen reader should announce that a balloon has been opened.
A blind person otherwise has no clue that something just happened.

### ❌ Actual result

No acoustic feedback.
A blind person does not know that a balloon has opened.

### ❓ Possible solution

Maybe the default behavior should be to focus the balloons content after using the shortcut.
Right now, the user needs to press [TAB] to set the focus to the balloon's toolbar. Before, they wouldn't really know something happened. Of course, this would be a UX / focus management change and should be discussed

## 📃 Other details

* Browser: Chrome
* OS: Mac OS
* First affected CKEditor version: Tested 41.3.1
* Reproducible via VoiceOver and JAWS

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by CKEditorBot** [comment]
There has been no activity on this issue for the past year. We've marked it as stale and will close it in 30 days. We understand it may still be relevant, so if you're interested in the solution, leave a comment or reaction under this issue.

**Comment by CKEditorBot** [comment]
We've closed your issue due to inactivity. We understand that the issue may still be relevant. If so, feel free to open a new one (and link this issue to it).
